mountpoint-s3/scripts/fs_latency_bench.sh

```diff
@@ -126,9 +126,9 @@ do
     dir_size=$(awk "BEGIN {print $dir_size*10}")
 done
 
-
-# start time to first byte benchmark
-jobs_dir=mountpoint-s3/scripts/fio/read_latency
+run_file_benchmarks() {
+  category=$1
+  jobs_dir=mountpoint-s3/scripts/fio/${category}_latency
 for job_file in "${jobs_dir}"/*.fio; do
   mount_dir=$(mktemp -d /tmp/fio-XXXXXXXXXXXX)
   job_name=$(basename "${job_file}")
```

Comment:
We should update indentation for this block. Sorry, I missed this part in the last review.